Api/Data/ProductEventInterface.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2017-2022 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Bolt product event data interface
+ * @api
+ */
+interface ProductEventInterface extends ExtensibleDataInterface
+{
+    public const ID = 'id';
+
+    public const PRODUCT_ID = 'product_id';
+
+    public const TYPE = 'type';
+
+    public const CREATED_AT = 'created_at';
+
+    /**
+     * Allowed product event operation types
+     */
+    public const TYPE_UNKNOWN = 'unknown';
```

Comment:
Why do we need unknown type?

Reply:
I think we are not going to use it, let's remove at least for now.

Reply:
Removed.